.github/workflows/ci-host.yml

```diff
@@ -11,12 +11,20 @@ jobs:
     steps:
       - name: Build the container
         run: |
-          DATE=`date "+%F"`
-          IMAGE="ghcr.io/rpm-software-management/dnf-ci-host"
+          # hack: Github replaces secrets with *** in the whole output (even in
+          # e.g. Copr URLs printed by rpm-gitoverlay). If there's a comment (#)
+          # at the end of the secret (e.g.  "rpm-sofware-management #"), this
+          # will clean it up and since it is no longer the whole secret being
+          # printed, Github won't hide it anymore.
+          GHCR_USER=${{secrets.GHCR_USER}}
 
-          echo ${{secrets.GITHUB_TOKEN}} | docker login ghcr.io -u rpm-software-management --password-stdin
+          if [ -n "$GHCR_USER" ]; then
```

Comment:
Why not just do an early `exit 0` instead of indenting the rest of the script into an `if`?

Reply:
I wasn't sure I can call `exit` in the workflow, but I see now its already used in the `setup-ci`. :+1: 

Changed it.